TaskModules/powershell/TaskModuleSqlUtility/SqlPackageOnTargetMachines.ps1

```diff
@@ -340,6 +349,37 @@ function LocateSqlPackageFromVSInstallationRoot {
     return $null, 0
 }
 
+function GetDacFxMajorVersion {
+    [CmdletBinding()]
+    Param (
+        [string] $path
+    )
+    try {
+        $file = [System.IO.Path]::Combine($sqlDacRoot, "Microsoft.SqlServer.Dac.dll")
+
+        Write-Verbose "Getting version info for file $file."
+
+        if (Test-Path -LiteralPath $file -PathType 'Leaf') {
+
+            $fileVersionInfo = (Get-Command -Name $file -ErrorAction 'Stop').FileVersionInfo
+            $fileVersion = $fileVersionInfo.FileVersion
```

Comment:
this is ok, alternative for getting sqlpackage version is sqlpackage /version

Reply:
I prefer just looking at some file properties over launching an executable